kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/firewall/FirewallAutoNatConfig.java

```diff
@@ -13,13 +13,16 @@
 package org.eclipse.kura.net.firewall;
 
 import org.eclipse.kura.net.NetConfig;
+import org.osgi.annotation.versioning.ProviderType;
 
 /**
  * Represents an automatic NAT configuration
  *
  * @author eurotech
```

Comment:
As you have previously deleted @authour tags in this PR, you can also remove this.